ccp/docker-compose.yml

```diff
@@ -31,6 +31,9 @@ services:
       BEAM_PROXY_URL: http://beam-proxy:8081
       RETRY_COUNT: ${FOCUS_RETRY_COUNT}
       EPSILON: 0.28
+      QUERIES_TO_CACHE: '/queries_to_cache.conf'
+    volumes:
+      - /etc/bridgehead/queries_to_cache.conf:/queries_to_cache.conf
```

Comment:
This would result in the creation of a folder, if the file does not exist already. Can we find a backwards compatible way? An optimization such as caching should not cause operational issues.

Reply:
Changed it to include the config file directly in the same repo and directory, better?

Reply:
This is on the same level as the ccp vars, right? I think this is absolutely fine!